ecoscope/contrib/geemap.py

```diff
@@ -0,0 +1,96 @@
+import ee
```

Comment:
Will need to copy over the current MIT license supplied by geemap. Take a look at the foliummap contrib module we have for the example.

Reply:
Done. I also moved out the code from __init.py__ to geemap and did away with importing the version and function from there so we have all geemap code in just one file as well, so we don't have to replicate the license info.